src/containers/Cfe/index.tsx

```diff
@@ -31,20 +34,70 @@ interface CfeProps {
     viewerHeader: { cellId: string; label: string; value: string };
 }
 
-class Cfe extends React.Component<CfeProps> {
+interface CfeState {
+    defaultActiveKey: string[];
+    dontShowSmallScreenWarningAgain: boolean;
+    openKeys: string[];
+    width: number;
+    currentTab: string;
+}
+
+class Cfe extends React.Component<CfeProps, CfeState> {
     private static panelKeys = ["groupings", "selections"];
-    public state = {
+
+    private static hackyViewerTitleOverlay = (id: string, label: string, value: string) => {
+        const parent = document.querySelector(".cell-viewer-wrapper .ant-layout-content");
+        let wrapper = document.querySelector(".image-info-wrapper");
+        if (parent) {
+            if (!wrapper) {
+                wrapper = document.createElement("div");
+                wrapper.className = "image-info-wrapper";
+            }
+            parent.appendChild(wrapper);
+            // FIXME: the CSS Modules class names here aren't working, probably
+            // need to turn them into regular class names and use global selectors
+            // in the stylesheet.
+            wrapper.innerHTML = `
+                <h4 className="viewer-overlay-title">
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/API/Element/className
Part of what I was trying to get at was that `className` is not what you use in plain html. You use className as the js property (correctly on line 54) but in the html it's supposed to be `class`.

Reply:
Oh! of course! 🙃